apps/web/src/services/master-lists.ts

```diff
@@ -31,10 +31,15 @@ export interface MilestoneType extends StyleOption {
 }
 
 // milestone status' for adding milestones
-export const useGetMilestoneOptions = (): MilestoneType[] => {
+export const useGetMilestoneOptions = (statusId: number): MilestoneType[] => {
   const { data: milestones } = useSWRImmutable('ienmaster/status', fetcher);
 
-  return milestones?.data.filter((item: { id: number }) => item.id == 10003)[0].children;
+  // All tab defaults to 10000,
```

Comment:
what does this comment mean?
I wonder if there is a case that `statusId` could be 10000.

Reply:
when a new applicant gets created, they have no 'default' status for the details page to go to, so it defaults to 10000 which is the index value of the 'All' tab in the Applicants Table page, this just prevents it from erroring out.  I suppose we could just change the default landing tab to Intake if theres not status'